src/pl/plr.spec

```diff
@@ -3,10 +3,14 @@ Name:           plr-hawq
 Version:        08.03.00.14
 Release:		0
 Prefix:         /usr/local
-License:		GPL
+License:		Apache
```

Comment:
I guess we need to move back to GPL which is exactly the license of PL/R. There has been a upstream PL/R spec file in which license is said to be BSD, but I suspect that is wrong.